rules/windows/defense_evasion_suspicious_scrobj_load.toml

```diff
@@ -13,7 +13,7 @@ Identifies scrobj.dll loaded into unusual Microsoft processes. This usually mean
 executed in the target process.
 """
 from = "now-9m"
-index = ["logs-endpoint.events.*"]
+index = ["logs-endpoint.events.library-*", "logs-windows.sysmon_operational-*"]
```

Comment:
Will need to add `windows` to integration metadata field if we are opening this up to windows integration as well.